reflectable/lib/src/builder_implementation.dart

```diff
@@ -4371,24 +4428,24 @@ int _declarationDescriptor(ExecutableElement element) {
   }
 
   if (element is PropertyAccessorElement) {
-    result = element.isGetter ? constants.getter : constants.setter;
+    result |= element.isGetter ? constants.getter : constants.setter;
     handleReturnType(element);
   } else if (element is ConstructorElement) {
     if (element.isFactory) {
-      result = constants.factoryConstructor;
+      result |= constants.factoryConstructor;
```

Comment:
It seems less error-prone to initialize `result` to zero and use `|=` everywhere, rather than (as previously) keeping an eye on which bit-setting operations occur first in the control flow, and using `=` on exactly those bit-setting operations.